pom-with-benchmarks.xml

```diff
@@ -67,7 +67,7 @@
 			<id>mvn-public</id>
```

Comment:
I am assuming that most of the changes are due to a merge from an older pom file on your side...